tests/post_training/test_templates/test_smooth_quant.py

```diff
@@ -249,9 +267,16 @@ def test_empty_stats(self, mocker, tmpdir):
         assert len(arg.transformations) == 2
 
         mm_metatype = self.get_matmul_metatype()
-        matmuls = [node for node in graph.topological_sort() if node.metatype == mm_metatype]
-        for transformation in arg.transformations:
-            assert self.get_target_node_name(transformation) != matmuls[0].node_name
```

Comment:
Why this assert and loop trough `arg.transformations` were removed?

Reply:
Because with torchFX transformations it is impossible to get target_node name (it is inside a closure). I refactored the code and now it checks the same things by mock backend functions